src/statd/python/yanger/yanger.py

```diff
@@ -156,6 +156,13 @@ def iface_is_dsa(iface_in):
         return False
     return True
 
+def iface_is_internal(iface_in):
+    if "group" not in iface_in:
+        return False
+
+    if iface_in["group"] != "internal":
+        return False
+    return True
```

Comment:
```suggestion
    return iface_in.get("group") == "internal"
```
`.get()` on dictionaries is pretty handy in these situations.